README.md

```diff
@@ -45,6 +45,9 @@ sky = SkyRocket:new({
 
   -- Which mouse button to hold to resize a window?
   resizeMouseButton = 'left',
+
+  -- Should the current window be focused & brought to the front when you click on it?
+  enableSelection = false,
```

Comment:
**chore (blocking):** Can you rename this parameter to `focusWindowOnClick`?

Rationale: `enableSelection` is not quite correct. You can always _select_ a window and manipulate it, but this parameter is only controlling whether to _focus_.

Reply:
Done!